tests/Microsoft.Bot.Connector.Tests/Authentication/ManagedIdentityAuthenticatorTests.cs

```diff
@@ -20,7 +20,14 @@ public class ManagedIdentityAuthenticatorTests
         private readonly Func<string, string> audience = (id) => $"audience {id} ";
 
         [Fact]
+
+/* Unmerged change from project 'Microsoft.Bot.Connector.Tests (net6.0)'
+Before:
```

Comment:
same here

Reply:
fixed, I removed all the auto generated comments